src/builtins/Transformers.jl

```diff
@@ -326,6 +326,56 @@ end
 MLJBase.inverse_transform(transformer::UnivariateStandardizer, fitresult, w) =
     [inverse_transform(transformer, fitresult, y) for y in w]
 
+##########################################################################################
+## CONTINUOUS TRANSFORM OF TIME TYPE FEATURES
+
+"""
+    UnivariateTimeTypeToContinuous(zero_time=nothing, step=Hour(24))
+
+Convert `Date`, `DateTime`, and `Time` vectors to `Float64` by assuming `0.0` corresponds
```

Comment:
Minor: I suggest "Convert **a** `Date`, `DateTime`, and `Time` **vector** to prevent confusion with multi-variate case.